customize.sh

```
@@ -1,47 +1,46 @@
 #!/system/bin/sh
 awk '{print}' "$MODPATH"/common/SuperMarioTweaker-Banner
-sleep 2
 ui_print ""
-sleep 1
+if [ -d "/data/adb/modules/DT"]; then
```

Shellcheck (0.10.0)

[error] Couldn't parse this test expression. Fix to allow more checks.

(SC1073)